modules/health_quest/app/controllers/health_quest/v0/patients_controller.rb

```diff
@@ -4,17 +4,21 @@ module HealthQuest
   module V0
     class PatientsController < HealthQuest::V0::BaseController
       def signed_in_patient
-        render json: factory.get.response[:body]
+        render json: factory.get(current_user.icn).response[:body]
       end
 
       def create
-        render json: factory.create.response[:body]
+        render json: factory.create({}).response[:body]
```

Comment:
Passing in blank params because we create the Patient resource from the logged-in current_user. Also, we don't want anyone passing any parameters in here during a POST call.